db/migrate/20230224151935_remove_authorize_net_c_ctoken_from_account.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAuthorizeNetCCtokenFromAccount < ActiveRecord::Migration[5.2]
+  def change
+    safety_assured { remove_column :accounts, :credit_card_authorize_net_payment_profile_token }
+  end
+end
```

Comment:
According to this: https://github.com/ankane/strong_migrations#removing-a-column

The app should be restarted to be sure this migration doesn't cause problems, def. we'll need ops assistance to deploy this. But to be sure it doesn't cause any problem, another PR with only the `self.ignored_columns = %i[credit_card_authorize_net_payment_profile_token]` added to the `Account` model should be merged and deployed first.

Reply:
https://github.com/3scale/porta/pull/3702